tests/storage/test_base.py

```diff
@@ -55,6 +55,8 @@ def runWithConnection(func, *args, **kwargs):  # type: ignore[no-untyped-def]
         engine = create_engine(sqlite_config)
```

Comment:
I'd *love* to test these with postgres too, but not sure if there's an easy way to do that. 😢

Reply:
What's the objection/blocker here, sorry? Would mocking out calls to psycopg2 suffice, or is the point that you want to assert what SQL gets executed?

Reply:
Just seems like a big faff to sometimes test against postgres and sometimes test against sqlite. I was having trouble figuring out how to properly structure the tests. I'll poke at it again.